### DISCUSSION TITLE
Simulate unplugging

### DISCUSSION THREAD

**Discussion by quenthal** [syssi/homeassistant-goecharger-mqtt#171] [Q&A]
Looking at the main page of this repo it there is no checkbox for enabled or supported in "Simulate unplugging permanently", and there is also note that it is not exposed in firmware 0.53.1 for mqtt.

Does it function for later firmwares? How to use it? As it is possibly exposed as switch, how to simulate the seconds that it is unplugged?

**Comment by syssi** [comment]
The setting (`sumd: simulate unpluging duration (in milliseconds)`) is exposed in 0.56.2:

## Key `sumd`

> /go-eCharger/072246/sumd 10000

| Key        | R/W        | Type                         | Category      | Description                                                                         |
| ---------- | ---------- | ---------------------------- | ------------- | ----------------------------------------------------------------------------------- |
| sumd       | R/W        | milliseconds                 | Config        | simulate unpluging duration (in milliseconds)                                       |

But does this help much? Could you ask the manufacturer how to use the feature?

**Reply by quenthal** [reply]
I can do that. Do you happen to know what is the difference between "simulate unplugging" and "simulate unplugging permanently" and are they available in latest version?

**Reply by syssi** [reply]
Are we talking about

```
| su         | R/W        | bool                         | Config        | simulateUnpluggingShort                                                             |
| sua        | R/W        | bool                         | Config        | simulateUnpluggingAlways                                                            |
``` 

The API keys (`su`, `sua`) aren't exposed via MQTT yet (56.9 beta).

**Reply by syssi** [reply]
I don't know the feature in detail. Any insights / additional details would be awesome!

**Reply by quenthal** [reply]
> Are we talking about
> 
> ```
> | su         | R/W        | bool                         | Config        | simulateUnpluggingShort                                                             |
> | sua        | R/W        | bool                         | Config        | simulateUnpluggingAlways                                                            |
> ```
> 
> The API keys (`su`, `sua`) aren't exposed via MQTT yet (56.9 beta).

Yes. I also got suggestions on setting these simulations on the app with claims that go-e would follow these settings. I haven't yet tried this with mqtt (for example activating eco) or forcing state on or off - is go-e then doing this even without manual command from mqtt. I'm not sure if someone already has tried this?
